workspace/apps/pidp/src/app/features/shell/pages/support-error/support-error.page.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+
+import { provideAutoSpy } from 'jest-auto-spies';
+
+import { SupportErrorPage } from './support-error.page';
+
+describe('SupportErrorPage', () => {
+  let component: SupportErrorPage;
+  let router: Router;
+
+  const mockActivatedRoute = {
```

Comment:
It's a bit deceptive having a mockActivatedRoute when there is no ActivatedRoute dependency.  I would pass an empty string directly to the expected test result, which forces the next developer to update it if the route changes.